File: Python/Intermediário 3.py
n1 = int ( input ())
n2 = int ( input ())
n3 = int ( input ())

if n1 > n2:
    Aux = n1
    n1 = n2
    n2 = Aux
if n1 > n3:
    Aux = n1
    n1 = n3
    n3 = Aux
if n2 > n3:
    Aux = n2
    n2 = n3
    n3 = Aux
    
print("%d"%n2)

File: C++/Intermediário 3.cpp
/*
    Autor : Sandeison Silva Fernandes
    Data: 01 / 06 / 2015

*/

#include <iostream>
#include <stdio.h>
using namespace std;

int main ( )
{
    int num_1, num_2,num_3, aux;
           cin >> num_1 >> num_2 >> num_3;
           while (( num_1 >num_2 ) || ( num_2 > num_3 )) {
              if (num_1 > num_2) {
                 aux = num_1;
                 num_1 = num_2;
                 num_2 = aux;
              }
              if (num_2 > num_3) {
                 aux = num_2;
                 num_2 = num_3;
                 num_3 = aux;
              }
           }
           cout <<  num_2 << endl;

    return 0 ;
}
